Userland/Libraries/LibJS/Runtime/Temporal/CalendarPrototype.cpp

```diff
@@ -288,7 +293,7 @@ JS_DEFINE_NATIVE_FUNCTION(CalendarPrototype::month_code)
     }
 
     // 5. Assert: temporalDateLike has an [[ISOMonth]] internal slot.
-    // NOTE: The assertion happens at iso_month() call.
+    // NOTE: The assertion happens in iso_month() call.
```

Comment:
This change belongs to the commit LibJS: Merge ISOMonthCode and BuildISOMonthCode
(The typo fix from at -> in)